js/ui/calendar/ui.calendar.js

```diff
@@ -1227,6 +1239,7 @@ const Calendar = Editor.inherit({
             case 'dateSerializationFormat':
             case 'cellTemplate':
             case 'showTodayButton':
+            case 'navigateToSameDay':
```

Comment:
```js
  case 'showTodayButton':
    this._invalidate();
    break;
  case 'navigateToSameDay':
    break;
```

There is no need to invalidate widget. The value of this option is processed during the calendar navigation, the appearance of the widget does not change.

Reply:
Thanks